Recurly/Pager.cs

```diff
@@ -40,16 +70,6 @@ public T Current
             }
         }
 
-        private void FetchNextPage()
-        {
-            var response = RecurlyClient.MakeRequest<Pager<T>>(Method.GET, Next);
-            var pager = response.Data;
-            // this is brittle
-            this.Next = pager.Next;
-            this.Data = pager.Data;
-            this.HasMore = pager.HasMore;
```

Comment:
I'm curious about the comment `this is brittle`. The logic is basically the same, but now it's inside the `Clone` method. Does putting the same code in a method make it less brittle? I don't see anything wrong with the way you have it now, I'm just trying to learn about the process here

Reply:
I said it was brittle because it you have to keep it to date and it might be needed in multiple spots. It's now at least in one place. So less brittle but could maybe automatically look at all the properties and clone them over.

Reply:
Right, that makes sense. I agree that it would be a bit more robust to copy all the properties rather than pulling specific ones, but I also think it would be unlikely that this would cause problems in the future. Thanks for clarifying